n2y/blocks.py

```diff
@@ -388,6 +392,44 @@ def to_pandoc(self):
         return Row(('', [], []), cells)
 
 
+class ColumnListBlock(Block):
+    def __init__(self, client, notion_data, page, get_children=True):
+        super().__init__(client, notion_data, page, get_children)
+
+    def to_pandoc(self):
+        cells = self.children_to_pandoc()
+        colspec = [(AlignDefault(), ColWidthDefault()) for _ in range(len(cells))]
+        table = Table(
+            ('', [], []),
+            Caption(None, []),
+            colspec,
+            TableHead(('', [], []), []),
+            [TableBody(
+                ('', [], []),
+                RowHeadColumns(0), [],
+                [Row(('', [], []), cells)])],
+            TableFoot(('', [], []), [])
+        )
+        return table
+
+
+class ColumnBlock(Block):
+    def __init__(self, client, notion_data, page, get_children=True):
```

Comment:
Why do you override the `__init__` here, since it doesn't seem to do anything different than `Block.__init__`.?

Reply:
left over from print statements